.eslintrc.json

```diff
@@ -17,7 +17,8 @@
   "globals": {
     "$": true,
     "chrome": true,
-    "moment": true
+    "moment": true,
+    "Chart": true
```

Comment:
What does this globals do? Allows the linter to recognize those keywords?

Reply:
Yes. Otherwise it will complain about it being undefined. With the exception of chrome, these could be `required` where needed and then gulp would just build them into the bundled code.  Not sure we'd want to do that yet or at all.